packages/testbed/src/app/pages/fx-settings/fx-settings.component.html

```diff
@@ -0,0 +1,16 @@
+@for (bus of buses; track bus) {
+<div class="mb-5">
+  <h2>
+    {{ bus.label }}
+    <span class="small text-muted"
+      >{{ withFxType(bus.fx.fxType$) | async }} ({{ bus.fx.fxType$ | async }})</span
```

Comment:
I don't think this `({{ bus.fx.fxType$ | async }})` are needed in the frontend... just if I can change the `fxType` with a number input, but if we're enabling the user to change the `fxType`, I rather go with a dropdown to choose the `fxType` or something like that

Reply:
but if this is just to show the usability, I'm fine with it

Reply:
The testbed application is just a simple app that showcases all features of the library. The main purpose is to give me a frontend where I can manually test the features.
The library exposes the FX type so I also added it to the testbed app. 🙂